registrations/views.py

```diff
@@ -125,15 +127,24 @@ def post(self, request, *args, **kwargs):
             )
             return redirect(reverse_lazy("registrations:confirm-clinic"))
 
-        send_registration_to_openhim.delay(
-            msisdn=session["registration_details"]["msisdn"],
-            referral_msisdn=session.get("registered_by"),
-            channel=session["channel"],
-            clinic_code=session["registration_details"]["clinic_code"],
-            persal=session.get("contact", {}).get("fields", {}).get("persal"),
-            sanc=session.get("contact", {}).get("fields", {}).get("sanc"),
-            timestamp=datetime.utcnow().timestamp(),
-        )
+        chain(
+            send_registration_to_rapidpro.s(
+                contact=session["contact"],
+                msisdn=session["registration_details"]["msisdn"],
+                referral_msisdn=session.get("registered_by"),
+                channel=session["channel"],
+                clinic_code=session["registration_details"]["clinic_code"],
+                timestamp=datetime.utcnow().timestamp(),
+            ),
+            send_registration_to_openhim.s(
+                referral_msisdn=session.get("registered_by"),
+                channel=session["channel"],
+                clinic_code=session["registration_details"]["clinic_code"],
+                persal=session.get("contact", {}).get("fields", {}).get("persal", None),
+                sanc=session.get("contact", {}).get("fields", {}).get("sanc", None),
+                timestamp=datetime.utcnow().timestamp(),
+            ),
+        ).apply_async()
```

Comment:
Do we need to worry about the result of `send_registration_to_rapidpro` going to `send_registration_to_openhim` over here?

Reply:
@rudigiesler nope. I've changed `send_registration_to_rapidpro` to return the msisdn. So that will be the first argument to `send_registration_to_openhim`